lymph/unilateral.py

```diff
@@ -127,27 +128,40 @@ def init_edges(self, graph):
         self.tumor_edges = []
         self.lnl_edges = []
         self.growth_edges = []
-
-        for (_, start_name), end_name in graph.items():
-            new_edge = Edge(
-                start=self.find_node(start_name),
-                end=self.find_node(end_name),
-            )
-            if new_edge.is_tumor_spread:
-                self.tumor_edges.append(new_edge)
-            elif new_edge.is_growth:
+        for (typ, start_name), end_names in graph.items():
+            if typ == 'lnl':
+                new_edge = Edge(
+                    start=self.find_node(start_name),
+                    end=self.find_node(start_name),
+                )
                 self.growth_edges.append(new_edge)
-            else:
-                self.lnl_edges.append(new_edge)
+            for end_name in end_names:
+                new_edge = Edge(
+                    start=self.find_node(start_name),
+                    end=self.find_node(end_name),
+                )
+                if new_edge.is_tumor_spread:
+                    self.tumor_edges.append(new_edge)
+                else:
+                    self.lnl_edges.append(new_edge)

```

Comment:
Nice catch of my mistake. Maybe the creation of the growth edges can be pulled out into another method at some point.